ci_framework/roles/cifmw_block_device/vars/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
`vars` directory is empty. We can drop this.

Reply:
done